ruby.rb

```diff
@@ -57,10 +59,10 @@ def update
     @article = Article.find_by_slug!(params[:slug])
 
     if @article.user_id == @current_user_id
```

Comment:
Have you heard of the code smell called `Tell Don't ask`. If not, google it. This is a violation of that rule. Move this logic into the article class. E.g:

```
def update

   if @article.update_attributes(article_params)
      render :show
   else
      render json: not_owned, status: :forbidden
   end
```